src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/NotificationsType.php

```diff
@@ -40,13 +40,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('show_notifs_new_orders', SwitchType::class, [
-                'required' => true,
+                'label' => $this->trans('Show notifications for new orders', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('This will display notifications when new orders are made in your shop.', 'Admin.Advparameters.Help'),
```

Comment:
@JevgenijVisockij @LouiseBonnard Don't you think that we could get rid of the following help texts? After reading the PR again, I feel like the label speaks for itself.

"This will display notifications when new orders are made in your shop."
"This will display notifications every time a new customer registers in your shop."
"This will display notifications when new messages are posted in your shop."

Reply:
Nice catch, @Julievrz, I'm all for it! We could even go further by making the information notification looks like in #22193 and, perhaps, by shortening the label names to sound less redundant, what do you think?

Reply:
I agree with your idea of making the information notification looks like in #22193. Regarding your second idea, maybe we could have a radio button that would allow the merchant to choose the notifications he/she wants to receive. Something like:
![image](https://user-images.githubusercontent.com/70583503/103783939-a239ff80-5039-11eb-8d25-53f047654e2a.png) 

If it is too long or complicated to implement, we could keep the current "Yes/No" buttons, by shortening the labels and putting "Show notifications for..." at the top. 

Not sure if I am being clear, let me know if you need me to specify something. 😅

Reply:
@LouiseBonnard I think this will be addressed in the issue I discussed this morning with Tristan, by implementing a toggle but this will be addressed in another issue as I think it is out of the scope of this PR.

Reply:
Sure, feel free to open a new issue to address this part if you prefer! We'll just have to prioritize it along with this one to make sure the user experience is fully improved on the page. ;-)